src/commands/reaction/index.ts

```diff
@@ -0,0 +1,7 @@
+import Google from "./Google";
+import Lmgtfy from "./Lmgtfy";
+
+export default [
```

Comment:
It seems `CreateServerEvent` was excluded here, not sure if intentional

Reply:
Accidental!